### ISSUE TITLE
Driver layout

### ISSUE THREAD

**Issue by Coreyboy1820** [libhal-google/libhal.github.io#112]
Comment where declaration vs. implementation is.

**Comment by kammce** [comment]
Can you explain in more detail what you mean by this?

**Comment by Coreyboy1820** [comment]
Yes, I think that in any driver you should specify where the declaration for a class is vs. the actual implementation. For example, possibly an // end of class name_of_class.

**Comment by kammce** [comment]
@Coreyboy1820 can you work this for the next sprint?
